bin/fif

```diff
@@ -14,7 +14,7 @@ fif() {
     echo "Need a string to search for!";
     return 1;
   fi
-  rg --files-with-matches --no-messages "$1" | fzf "$FZF_PREVIEW_WINDOW" --preview "rg --ignore-case --pretty --context 10 '$1' {}"
+  rg --files-with-matches --no-messages "$1" | fzf $FZF_PREVIEW_WINDOW --preview "rg --ignore-case --pretty --context 10 '$1' {}"
```

Comment:
Please add
```
  # if FZF_PREVIEW_WINDOW is undefined, quoting it breaks the script
  # shellcheck disable=SC2086
```
before the `rg --files-with-matches ...` line so that `shellcheck` doesn't complain about the removed quotes and the superlint checks pass.

Thanks

Reply:
Otherwise when I edit it next after a few months pass I'll reintroduce the bug because of humoring `shellcheck`.

Reply:
Sure! 🤦 I would had to anticipate that shellcheck would probably make the pipeline red, as the quotes were introduced to make him happy, hehe. 

Done! I've force-pushed, in case you don't want to squash the merge :)